### DISCUSSION TITLE
Wiki should be updated with this new css

### DISCUSSION THREAD

**Discussion by uaevuon** [mbnuqw/sidebery#1897] [General]
After FF133, current css files under **completely hide native tabs strip** on wiki do not work.

Only this [css](https://mrotherguy.github.io/firefox-csshacks/?file=autohide_tabstoolbar_v2.css) is needed.

**Comment by riveravaldez** [comment]
In case it's relevant: on updated Firefox release, setting vertical tabs and up-left button to "hide/show" allows for Sidebery to be the only visible vertical tabs interface, as long as the user resizes Firefox vertical tabs to zero size.
This is how it looks:
![e1986bc347ccf744](https://github.com/user-attachments/assets/dea68180-fd14-4ad5-9063-c145287dc25b)
